### DISCUSSION TITLE
Can't get generated setter to work.

### DISCUSSION THREAD

**Discussion by borisgoro** [xaviergonz/mobx-keystone#226] [Q&A] [answered]
I pasted the example code
```
@model("myCoolApp/Todo")
export class Todo extends Model({
  text: prop<string>({ setter: true }),
  done: prop(false, { setter: true }),
}) {}
```
into my source, but I get ```No overload matches this call``` on the first prop, and ```Expected 0-1 arguments, but got 2``` on the second. In ```/packages/lib/src/modelShared/prop.ts``` I see type support for setters but no ```prop<>``` overloads for them. What am I missing?

**Comment by xaviergonz** [comment, marked as answer]
It is now prop().withSetter(), but I forgot to change the docs. Will do so

**Reply by xaviergonz** [reply]
The docs should be OK now

**Reply by borisgoro** [reply]
Thank you! Just getting my feet wet, but the library looks great so far 👊
